types/binary_literal.go

```diff
@@ -183,6 +183,11 @@ func NewBitLiteral(s string) (BitLiteral, error) {
 	return BitLiteral(b), nil
 }
 
+// ToString implement ast.BinaryLiteral interface
+func (b BitLiteral) ToString() string {
+	return string(b)
```

Comment:
```suggestion
	return BinaryLiteral(b).ToString()
```

to ensure the implementations are the same.